src/main.js

```diff
@@ -33,6 +34,7 @@ Vue.use(Vuelidate)
 Vue.use(VueClipboard)
 Vue.use(InfiniteScroll)
 Vue.use(AsyncComputed)
+Vue.use(DynamicReactiveRefs)
```

Comment:
👀

Reply:
makes the refs reactive so we can trigger logic on $refs.actionModal being available

Reply:
thank you!